### ISSUE TITLE
Hardware configuration used to achieve results obtained in paper?

### ISSUE THREAD

**Issue by agenius5** [google-research/pegasus#147]
HI, @JingqingZ 

I have two queries.

1. What hardware configuration did you used for fine-tuning on various datasets to obtain results published in paper (especially amount of memory and which GPU) ?
2. How often should I save checkpoint (to evaluate of validation set) when fine-tuning on full datasets? Saving checkpoint too often (every 50 or 100 steps is time and memory consuming and by saving less I might miss the best validation results)

**Comment by JingqingZ** [comment]
I can't recall the exact setting but it was TPUv3.

Save per 1k step should be fine. The performance won't fluctuate much after sufficient steps.

**Comment by agenius5** [comment]
Alright, thanks!
